### ISSUE TITLE
Documents IDs overlap with train/test split

### ISSUE THREAD

**Issue by KennethEnevoldsen** [UniversalDependencies/UD_Danish#7]
~~The document IDs overlap between the training and test set for Danish. @dan-zeman I think the best solution would be to recreate the test set as this is a leakage that would lead to inflated performance on the test set. Is creating a new split an option? I will try to do it such that it follows the split of CDT.~~

**Comment by KennethEnevoldsen** [comment]
Sorry for this it seems like I spoke too soon. After double checking it seems like I made a mistake

But the test set DDT is in the training set for the CDT, which is frustrating, will try to see if I can fix this.

**Comment by dan-zeman** [comment]
> But the test set DDT is in the training set for the CDT, which is frustrating, will try to see if I can fix this.

It would obviously be better to to synchronize the data splits of the two treebanks, that is, if other things do not stand in the way. (UD has a strong recommendation that dev and test have at least 10K words each, if the treebank size permits it.)

**Comment by KennethEnevoldsen** [comment]
I have created an issue here:
https://huggingface.co/datasets/alexandrainst/dacoref/discussions/1

CDT itself sadly does not allow for issues
